workspace_host/interface.js

```diff
@@ -719,3 +734,51 @@ function destroySequence(workspace_id, res) {
         }
     });
 }
+
+function gradeSequence(workspace_id, res) {
+    var workspace_dir = `${workspacePrefix}/${id_workspace_mapper[workspace_id].localName}/project`;
+    var graded_file_list = id_workspace_mapper[workspace_id].settings.workspace_graded_files.map((file) => {return path.join(workspace_dir, file);});
+    graded_file_list = ['starter_code.c', 'starter_code.h'].map((file) => {return path.join(workspace_dir, file);});    // since in the DB the workspace_graded_file col is empty, hardcode file list
```

Comment:
> // since in the DB the workspace_graded_file col is empty

Hmm not sure what this means? `demo/workspace/{vscode,jupyterlab,xtermjs}` all have `gradedFiles` synced to the DB.

Reply:
Oh, that's just for my own testing purposes. I forgot to remove it.